info.go

```diff
@@ -147,6 +150,9 @@ func newProgramInfoFromFd(fd *sys.FD) (*ProgramInfo, error) {
 			runCount:        info.RunCnt,
 			recursionMisses: info.RecursionMisses,
 		},
+		jitedSize:            info.JitedProgLen,
+		loadTime:             time.Duration(info.LoadTime),
```

Comment:
Is there a risk of wrong information, as Go's `time.Duration` is of type `int64` while `LoadTime` from the kernel is of type `uint64`?

Reply:
Ah, I suppose there could be a risk of it overflowing, although that would mean the program has been loaded for over ~290 years. 😅
Would this be an okay assumption that programs wouldn't be loaded for that long?

Reply:
Don't know if uptime contests are still a thing, but I guess we'll find out in 290 years! ;) Of course this is fine.